esdb/persistent_options.go

```diff
@@ -178,8 +168,6 @@ type ListPersistentSubscriptionsOptions struct {
 	Authenticated *Credentials
 	// A length of time to use for gRPC deadlines.
 	Deadline *time.Duration
-	// Requires the request to be performed by the leader of the cluster.
-	RequiresLeader bool
```

Comment:
I'm not sure if all of these operations require the leader by default. Let me know if I should revert any.

Reply:
They do!